src/workloads/execution/CreateIndexSharded.yml

```diff
@@ -0,0 +1,146 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+
+Actors:
+- Name: InsertData
```

Comment:
Can you put this stage after creating the hashed index and sharding the collection? Otherwise the data will all start on one shard and need to be balanced while the indexes are being built, which isn't really what this workload was for. That should also speed up building the hashed index, since there won't be any data yet.